plugin/plugin_settings.py

```diff
@@ -117,7 +117,7 @@ def __update_settings_from_project_if_needed(self):
         self.__load_vars_from_settings(settings_handle, Settings.PREFIXES)
         log.debug(" All overrides applied.")
 
-    def __load_vars_from_settings(self, settings_handle, prefixes=[""]):
+    def __load_vars_from_settings(self, settings_handle, prefixes=None):
```

Comment:
On the other hand, it could just be a boolean called, for example, `read_project_settings` and the method would just use iterate PREFIXES directly if it is true. I think the flexibility of passing any list is not really needed. :)